src/site/content/en/reliable/resilient-search-experiences/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Resilient search experiences
+authors:
+  - demianrenzulli
+  - jeffposnick
+date: 2020-07-19
```

Comment:
```suggestion
date: 2020-06-19
```
@demianrenzulli I'm assuming you meant to set this date to today, not a month from today ;)